README.md

```diff
@@ -1,45 +1,56 @@
 ﻿# Reactive.Annex
 
-`Reactive.Annex` contains extension methods to help developers make the most out of the Reactive world.
-`Reactive.Annex.Uno` also contains an easy-to-use implementation of a dispatcher scheduler following the `IScheduler` interface.
+New types and extension methods built on top of [`System.Reactive`](https://github.com/dotnet/reactive).
 
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
 ![Version](https://img.shields.io/nuget/v/Reactive.Annex?style=flat-square)
 ![Downloads](https://img.shields.io/nuget/dt/Reactive.Annex?style=flat-square)
 
 ## Getting Started
 
-Add the `Reactive.Annex` nuget package and its dependencies to your project.
+1. Add the `Reactive.Annex` nuget package to your project.
 
-Then, make sure you are using the `System.Reactive.Linq` and `System.Reactive.Concurrency` namespaces in your file.
+1. Then, make sure you are using the `System.Reactive.Linq` and `System.Reactive.Concurrency` namespaces in your file.
```

Comment:
is this also meant to be 1. or 2.

Reply:
`1.` is good.
You use `1.` everywhere in markdown and the actual numbers are generated automatically when rendered.